argoverse/map_representation/map_api.py

```diff
@@ -85,6 +89,12 @@ def __init__(self) -> None:
             self.city_to_lane_bboxes_dict[city_name] = lane_bboxes
             self.city_to_da_bboxes_dict[city_name] = da_bboxes
 
+    @property
+    def map_files_root(self) -> Path:
+        if self.root is None:
+            raise ValueError("Root cannot be None!")
```

Comment:
maybe "Map root directory cannot be None!" is clearer?

Reply:
👍